src/seedu/addressbook/ui/Formatter.java

```diff
@@ -63,4 +67,14 @@ private static String getIndexedListItem(int visibleIndex, String listItem) {
         return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
     }
 
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    public String getPrintableString(Printable... printables) {
```

Comment:
good job implementing the interface.  call the method somewhere in some part of the addressbook so that you can actually see the output :p